dwave/cloud/cli.py

```diff
@@ -292,6 +323,18 @@ def _config_create(config_file, profile, ask_full=False):
     if profile != default_section and not config.has_section(profile):
         config.add_section(profile)
 
+    # set using Leap API
+    try:
+        sapi_token = _fetch_sapi_token(config_file, profile) if auto_token else None
+    except Exception as error:
+        click.echo(f"Failed to fetch SAPI token from Leap API ({error!s}).")
```

Comment:
Probably not this line but if I run `dwave auth login --profile test` and click `Decline` I get the error message `The provided authorization grant or refresh token is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client`, which is incorrect. Corner case that can be an issue for later, not in the least bit urgent.

Reply:
Good catch.